src/content/app/species/components/species-title-area/SpeciesTitleArea.tsx

```diff
@@ -122,7 +130,10 @@ const SpeciesTitleArea = () => {
           <h1 className={styles.speciesName}>{getDisplayName(species)}</h1>
           <span className={styles.assemblyName}>{species.assembly.name}</span>
         </div>
-        <div className={styles.assemblyCountWrapper}>
+        <div
+          className={styles.assemblyCountWrapper}
+          onClick={navigateToAssemblies}
+        >
```

Comment:
Either this `div`, or the Assemblies `span` (depending on which element is expected to be clickable) should instead become a `button`.

_(Personally, I would only expect the `Assemblies` to be clickable; but of course if Andrea said otherwise, stick with that.)_

Reply:
Now the InfoPill has collapsed again...

![image](https://github.com/Ensembl/ensembl-client/assets/6834224/19a28e2b-82bb-4076-87d0-bc8314f501bd)

I think we should replace the default padding of InfoPill with `2px 8px` instead of `0 8px`.